gui_qt/user_copy.py

```
@@ -159,6 +191,66 @@ def format_doctor_warning_fact(warning: str) -> str:
+    message = primary_recommendation_message(recommendation_facts)
+    if message in ready_messages:
+        return False
+    return bool(recommendation_facts)
+
+
+def primary_recommendation_message(recommendation_facts: list[str]) -> str:
+    """Map the first rendered recommendation fact to a short summary message."""
+    if not recommendation_facts:
+        return ""
+    first = recommendation_facts[0].strip()
+    if not first.startswith("建议："):
```

Ruff (0.15.20)

[warning] String contains ambiguous `：` (FULLWIDTH COLON). Did you mean `:` (COLON)?

(RUF001)